src/QMCHamiltonians/MPC.cpp

```diff
@@ -327,9 +315,7 @@ void MPC::initBreakup()
 
 std::unique_ptr<OperatorBase> MPC::makeClone(ParticleSet& qp, TrialWaveFunction& psi)
 {
-  // return new MPC(qp, Ecut);
-  std::unique_ptr<MPC> newMPC = std::make_unique<MPC>(*this);
-  newMPC->resetTargetParticleSet(qp);
```

Comment:
Why no longer needed?

Reply:
The current implementation is no-op at line 33
```
resetTargetParticleSet(qp) {}
```
and resetTargetParticleSet should go away eventually.